code/obj/item/gun/energy.dm

```diff
@@ -1300,26 +1300,24 @@ TYPEINFO(/obj/item/gun/energy/lawbringer)
 		..()
 
 	attack_hand(mob/user)
-		if (!owner_prints)
-			boutput(user, SPAN_ALERT("[src] has accepted your DNA string. You are its owner!"))
-			assign_name(user)
+		assign_name(user)
 		..()
 
 
 	//if it has no owner prints scanned, the next person to attack_self it is the owner.
 	//you have to use voice activation to change modes. haha!
 	attack_self(mob/user as mob)
 		src.add_fingerprint(user)
-		if (!owner_prints)
-			boutput(user, SPAN_ALERT("[src] has accepted your DNA string. You are its owner!"))
-			assign_name(user)
-		else
-			boutput(user, SPAN_NOTICE("There don't seem to be any buttons on [src] to press."))
+		assign_name(user)
 
 	proc/assign_name(var/mob/M)
+		if (owner_prints)
+			boutput(M, SPAN_NOTICE("There don't seem to be any buttons on [src] to press."))
```

Comment:
PR makes sense buut this message would go off every time you pick it up as written I think?

Reply:
Oops. Yeah.